src/addons/badges/badge-lazy.module.ts

```diff
@@ -0,0 +1,32 @@
+// (C) Copyright 2015 Moodle Pty Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { AddonBadgesIssuedBadgePage } from './pages/issued-badge/issued-badge';
+
+const routes: Routes = [
+    {
+        path: ':badgeHash',
+        component: AddonBadgesIssuedBadgePage,
+    },
```

Comment:
If this page is opened without split view, then the following code shouldn't be executed in the page:

```
const source = CoreRoutedItemsManagerSourcesTracker.getOrCreateSource(
    AddonBadgesUserBadgesSource,
    [this.courseId, this.userId],
);

this.badges = new CoreSwipeNavigationItemsManager(source);
```
This code is used to allow changing the current badge by swiping left/right when there is a spit view. Since there is no split view in this case, the code shouldn't be used.

I guess the best way to detect this is to use the "data" property of the route to identify it has split view or not, something like:

```suggestion
    {
        path: ':badgeHash',
        component: AddonBadgesIssuedBadgePage,
        data: { usesSplitView: false },
    },
```

Please notice you will need to add "usesSplitView: true" in the routes that use split view. 

If you search "discussionsPathPrefix" in the app you will see an example of how to use this route data.

Reply:
I've used the name `usesSwipeNavigation", because swipe navigation it's also used without the split view.